app/lib/aether/aether.coffee

```diff
@@ -132,6 +132,8 @@ module.exports = class Aether
         @pure = token.src
         @ast = token.ast
     else
+      if @language.id in ['cpp', 'java']
+        console.error('C++ code cannot be transpiled client side.')
```

Comment:
Do we want to hard stop here?

Reply:
ie throw and error or return or something?

Reply:
Yeah. Changed to an actual error, and decreased scope to C++. Great point!